lib/components/BraidProvider/BraidProvider.tsx

```diff
@@ -1,5 +1,11 @@
-import React, { createContext, useContext, ReactNode } from 'react';
-import { TreatProvider } from 'sku/react-treat';
+import React, {
+  createContext,
+  useContext,
+  ReactNode,
+  ComponentType,
+  AnchorHTMLAttributes,
+} from 'react';
+import { TreatProvider } from 'sku/treat';
```

Comment:
This looks like an accidental change, should be importing from `sku/react-treat`.

Reply:
😫

Reply:
This looks like a failed merge from master on my part.